website/profile/views.py

```diff
@@ -408,6 +409,44 @@ def oauth_application_detail(auth, **kwargs):
     return {"app_list_url": '',
             "app_detail_url": app_detail_url}
 
+@must_be_logged_in
+def personal_access_token_list(auth, **kwargs):
+    """Return token creation page with list of known tokens. API is responsible for tying list to current user."""
+    token_list_url = api_v2_url("tokens/")
+    return {
+        "token_list_url": token_list_url
+    }
+
+@must_be_logged_in
+def personal_access_token_register(auth, **kwargs):
+    """Register an API personal token: blank form view"""
```

Comment:
It's not strictly [OAuth2](http://tools.ietf.org/html/rfc6749#section-4.1) either, but I'll update the docstring.